src/components/GroupedList/GroupedList.tsx

```diff
@@ -110,6 +110,24 @@ export class GroupedList extends React.Component<IGroupedListProps, IGroupedList
     this._forceListUpdates();
   }
 
+  public onToggleCollapseAll(allCollapsed: boolean) {
```

Comment:
it is really weird to expose n event handler as a public method here like this. If you're making a public verb method, it should be something more like toggleCollapseAll.

And it should be part of IGroupedList interface defined in props file, which GroupedList should implement. It will get documented then.